<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Onboarding_test_tenant-20231219114036" tests="1" file=".py" time="16.129" timestamp="2023-12-19T11:40:52" failures="0" errors="1" skipped="0">
	<testcase classname="Onboarding_test_tenant" name="test_tenat_onboarding" time="16.129" timestamp="2023-12-19T11:40:52" file="Onboarding.py" line="36">
		<error type="TypeError" message="CUITGenerator.generate_random_cuit() takes 1 positional argument but 2 were given"><![CDATA[Traceback (most recent call last):
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\Onboarding.py", line 106, in test_tenat_onboarding
    cuit = CUITGenerator.generate_random_cuit(self.driver, error_css_selector )
TypeError: CUITGenerator.generate_random_cuit() takes 1 positional argument but 2 were given
]]></error>
		<system-out><![CDATA[El texto obtenido es: DOCUMENTACIÓN
El texto obtenido es: Para poder continuar al siguiente paso, será obligatorio completar los campos que se muestran a continuación.
]]></system-out>
	</testcase>
</testsuite>
